config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
+  root 'welcome#index'
   get "/", to: "application#welcome"
 
+  post "/applications", to: "applications#create"
+  get "/applications/new", to: "applications#new"
+  get "/applications/:app_id", to: "applications#show", as: "application"
+  patch "/applications/:app_id", to: "applications#update"
+  patch "/applications/add_pet/:app_id", to: "applications#add_pet"
```

Comment:
Here is where I think you probably could have created a `pet_applications` controller and done something like
```
patch "/admin/applications/:id", to: "pet_applications#update"
```
which will keep your routes RESTful and lean on join table you created. You would also have to change the url on the form.